prow/cluster/ghproxy.yaml

```diff
@@ -73,3 +73,19 @@ spec:
         effect: "NoSchedule"
       nodeSelector:
         dedicated: "ghproxy"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: default
+  labels:
+    app: ghproxy
+  name: ghproxy
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    app: ghproxy
+  type: NodePort
```

Comment:
Is there a reason we'd want this exposed outside the cluster?

Reply:
Added a TODO, would like not to change this in this PR

Reply:
If we don't need this outside the cluster, then we should change this to `type: ClusterIP`